src/overlays/actors/ovl_En_Reeba/z_en_reeba.h

```diff
@@ -15,17 +15,17 @@ typedef struct EnReeba {
     /* 0x01FC */ Vec3s morphTable[18];
     /* 0x0268 */ char unk_268[0x4];
     /* 0x026C */ EnReebaActionFunc actionfunc;
-    /* 0x0270 */ s16 unk_270;
-    /* 0x0272 */ s16 unk_272;
-    /* 0x0274 */ s16 unk_274;
-    /* 0x0276 */ s16 unk_276;
-    /* 0x0278 */ s16 unk_278;
+    /* 0x0270 */ s16 bigLeeverTimer; // big Leever uses to close and make distance from Link.
+    /* 0x0272 */ s16 surfaceTimer;
+    /* 0x0274 */ s16 sfxTimer; // delay between move sounds. 2x if big Leever.
```

Comment:
```suggestion
    /* 0x0274 */ s16 sfxTimer; // delay between moving sfx. 2x if big Leever.
```

Reply:
Im not against this change, but I also think we dont need to be this strict about referring to sfx as sfx everywhere. high level in this context "sound" is fine too imo